Samples/CHSHGame/Game.qs

```diff
@@ -0,0 +1,177 @@
+	// share an entangled qubit pair, they can measure their qubits and use the
+	// results in a way which enables them to win an incredible 85% of the
+	// time! This conclusively disproves the existence of local hidden
+	// variables, because if the entangled qubits were carrying along with them
+	// some information (encoded as a string of bits) then Alice and Bob could
+	// have pre-shared that same information to help in their classical
+	// strategy. However, because no string of bits exists which can improve
+	// the classical strategy beyond a 75% success rate, there cannot
+	// be such a string of bits inside the entangled qubits which enables the
+	// quantum strategy to work so well: it must be something else, something
+	// spookier!
+    
+    //////////////////////////////////////////////////////////////////////////
+    // This Program //////////////////////////////////////////////////////////
+    //////////////////////////////////////////////////////////////////////////
+
+	// This program implements the CHSH quantum strategy, which is then run
+	// thousands of times to compare its success rate against the classical
+	// strategy. The quantum success rate converges to around 0.85 (actually
+	// cos^2(pi/8)) while the classical success rate converges to 0.75.
+
+	// The quantum strategy involves Alice and Bob measuring their qubits in
+	// various bases depending on their input bits. Alice measures her qubit
+	// in the Z basis if she is given a 0, and the X basis if she is given a 1.
+	// Bob measures his qubit in similar bases, but rotated pi/8 radians around
+	// the unit circle. This measurement scheme ensures Alice and Bob have an
+	// 85% probability of their qubits collapsing to the same value, except
+	// when both X and Y are 1, in which case they have an 85% probability of
+	// their qubits collapsing to *different* values - thus satisfying the 
+	// X * Y == A xor B formula with an 85% probability in all cases. This
+	// strategy works regardless of who first measures their qubit.
+
+	// Measurement in Bob's nonstandard bases is accomplished by first rotating
+	// the state vector by pi/8 radians in one direction or another, then
+	// measuring in the standard computational (Z) basis.
+
+    operation MeasureAliceQbit(bit : Bool, qubit : Qubit) : (Result)
+    {
+        body
+        {
+            if (bit)
+            {
+                // Measure in sign basis if bit is 1
+                return Measure([PauliX], [qubit]);
+            }
+            else
+            {
+                // Measure in computational basis if bit is 0
+                return Measure([PauliZ], [qubit]);
+            }
+        }
+    }
+
+    operation MeasureBobQbit(bit : Bool, qubit : Qubit) : (Result)
+    {
+        body
+        {
+            if (bit)
+            {
+                // Measure in -pi/8 basis if bit is 1
+                Ry(0.78539816339, qubit);
```

Comment:
Nit: I would suggest to use constant `Microsoft.Quantum.Extensions.Math.Pi` here instead of a number. Not that I expect the value of Pi to change in the foreseeable  future... :-)